drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -216,6 +216,12 @@ def dob
       object.dob if current_permission?(permission: :can_view_dob, entity: object)
     end
 
+    def first_name
+      return 'Anonymous' if object.first_name.blank? && object.last_name.blank?
```

Comment:
I wonder if representing anonymous clients is more of a front-end interface concern than something for the API to handle.

One thought, could the 'Anonymous' string be used to populate a form and then end up being persisted back to the client record? That seems like it could have undesirable side-effects

Reply:
Hm yeah, thats a really good point. I'll move it to the frontend so its display only.